### ISSUE TITLE
Bad Gateway when exporting with COCO and YOLO

### ISSUE THREAD

**Issue by MiguelFreire** [heartexlabs/label-studio#1236]
Hello,

I have label-studio behind a NGINX reverse proxy. I'm trying to export a 230 sample dataset with COCO and YOLO format but I'm getting a bad gateway error. All the other export methods work without any issue.

Any clue if this comes from your web API or the nginx reverse proxy? If so any suggestion on how I can fix the issue?

Cheers,
Miguel

**Comment by makseq** [comment]
@MiguelFreire Hello, 

The export might be slow because of image downloading (YOLO and COCO put images to output archive). NGINX might reset the connection before this download finishes.

**Comment by MiguelFreire** [comment]
I tried to increase timeout to 5mim and the error became a "Bad Gateway". Should I also increase it in your NGINX service?

Cheers

**Comment by makseq** [comment]
@MiguelFreire yes, nginx must he increased too. 
How many tasks do you have?

**Comment by makseq** [comment]
@MiguelFreire Please, have a look to the latest master, I've implemented an api call without image downloading step, it had to speed up this process. Just call 
```
/api/projects/{id}/export?exportType=COCO&download_resources=True/False
```

**Comment by smoreface** [comment]
Fixed with [version 1.3.0 release](https://github.com/heartexlabs/label-studio/releases/tag/v1.3.0).
